edx_sga/static/js/src/edx_sga.js

```diff
@@ -1,44 +1,49 @@
 /* Javascript for StaffGradedAssignmentXBlock. */
 function StaffGradedAssignmentXBlock(runtime, element) {
     function xblock($, _) {
-        var uploadUrl = runtime.handlerUrl(element, 'upload_assignment');
-        var downloadUrl = runtime.handlerUrl(element, 'download_assignment');
-        var annotatedUrl = runtime.handlerUrl(element, 'download_annotated');
-        var getStaffGradingUrl = runtime.handlerUrl(element, 'get_staff_grading_data');
-        var staffDownloadUrl = runtime.handlerUrl(element, 'staff_download');
-        var staffAnnotatedUrl = runtime.handlerUrl(element, 'staff_download_annotated');
-        var staffUploadUrl = runtime.handlerUrl(element, 'staff_upload_annotated');
-        var enterGradeUrl = runtime.handlerUrl(element, 'enter_grade');
-        var removeGradeUrl = runtime.handlerUrl(element, 'remove_grade');
-        var template = _.template($(element).find("#sga-tmpl").text());
-        var gradingTemplate;
+        var uploadUrl = runtime.handlerUrl(element, 'upload_assignment'),
+        downloadUrl = runtime.handlerUrl(element, 'download_assignment'),
+        annotatedUrl = runtime.handlerUrl(element, 'download_annotated'),
+        getStaffGradingUrl = runtime.handlerUrl(
+          element, 'get_staff_grading_data'
+        ),
+        staffDownloadUrl = runtime.handlerUrl(element, 'staff_download'),
+        staffAnnotatedUrl = runtime.handlerUrl(
+          element, 'staff_download_annotated'
+        ),
+        staffUploadUrl = runtime.handlerUrl(element, 'staff_upload_annotated'),
+        enterGradeUrl = runtime.handlerUrl(element, 'enter_grade'),
+        removeGradeUrl = runtime.handlerUrl(element, 'remove_grade'),
+        template = _.template($(element).find('#sga-tmpl').text()),
+        gradingTemplate;
```

Comment:
On second thought, these actually seem a lot more readable as separate `var` statements... Sorry. What do you think @carsongee ?